src/tool_operate.c

```diff
@@ -2328,6 +2328,16 @@ static CURLcode transfer_per_config(struct GlobalConfig *global,
       else {
         result = FindWin32CACert(config, tls_backend_info->backend,
                                  "curl-ca-bundle.crt");
+#if defined(USE_WIN32_CRYPTO) && \
+    !defined(CURL_WINDOWS_OPENSSL_NEVER_AUTOMATIC_USE_WINDOWS_CA_STORE) && \
+    !defined(LIBCURL_WINDOWS_OPENSSL_NEVER_USE_WINDOWS_CA_STORE)
```

Comment:
I don't think the `CURL_WINDOWS_OPENSSL_NEVER_AUTOMATIC_USE_WINDOWS_CA_STORE` is necessary either

Reply:
I remove it